### PR TITLE
Part of #17670 Replace Typography with Text component for 5 files.

### PR DESCRIPTION
**Pull request by PrgrmrHarshShukla**
## Explanation
In this PR I have worked on the issue #17670 Replace Typography with Text component and issue #18714 Replace deprecated design system typography consts with enums for the files:

1. ui/components/app/nfts-tab/nfts-tab.js
2. ui/components/app/snaps/copyable/copyable.js
3. ui/components/app/snaps/snap-remove-warning/snap-remove-warning.js
4. ui/pages/keychains/restore-vault.js
5. ui/pages/swaps/fee-card/fee-card.js

### Before
**Could not find the other components in the UI.
Please help.**

Screenshot (85)

### After

Screenshot (87)

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: ui/components/app/snaps/copyable/copyable.js [hunk 1]
```diff
@@ -30,14 +29,14 @@ export const Copyable = ({ text }) => {
       paddingTop={2}
       paddingBottom={2}
     >
-      <Typography
-        variant={TypographyVariant.H6}
+      <Text
+        variant={TextVariant.H6}
```
Comment (georgewrmarshall) [inline ui/components/app/snaps/copyable/copyable.js:33]: incorrect TextVariant

File: ui/pages/keychains/restore-vault.js [hunk 2]
```diff
@@ -116,14 +110,10 @@ class RestoreVaultPage extends Component {
...
                 </Button>,
               ])}
-            </Typography>
-            <Typography
-              color={TextColor.textDefault}
-              margin={0}
-              marginBottom={4}
-            >
+            </Text>
+            <Text color={TextColor.textDefault} margin={0} marginBottom={4}>
```
Comment (georgewrmarshall) [inline ui/pages/keychains/restore-vault.js:114]:
```suggestion
            <Text color={TextColor.textDefault} marginBottom={4}>
```

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
hey @PrgrmrHarshShukla, Left some suggestions could you please add screenshots to the PR description

**Comment by PrgrmrHarshShukla** [comment]
Hello @georgewrmarshall ,

I added the screenshots of the component I was able to find in the UI.
Kindly help for the other components.

**Comment by georgewrmarshall** [comment]
Hey @PrgrmrHarshShukla, the best way is to follow @brad-decker's comment here https://github.com/MetaMask/metamask-extension/pull/18741#issuecomment-1538837547

**Comment by georgewrmarshall** [comment]
The other option you have for gathering before / after screenshots is to create a story for the component. Docs are here https://metamask.github.io/metamask-storybook/?path=/story/getting-started-documentation-guidelines--page#creating-a-story

```jsx
import React from 'react';
import MyComponent from '.';

export default {
  title: 'Components/UI/MyComponent', // title should follow the folder structure location of the component. Don't use spaces.

};

export const DefaultStory = () => <MyComponent />;

DefaultStory.storyName = 'Default';
```

**Comment by georgewrmarshall** [comment]
Hey @PrgrmrHarshShukla, just wanted to check in. It looks like this PR is almost there. Did you still want to work on it? If so, that's great! Let us know if you need any assistance or if there are any blockers. If you're unable to continue working on it or haven't had a chance to address it, no worries! I can take it from here and carry it forward. Cheers!

**Comment by georgewrmarshall** [comment]
Hey @PrgrmrHarshShukla, thanks for all your effort with this PR I am going to close it as it has become stale. Please feel free to open another PR in future. Thanks for your contributions thus far!

**Comment by PrgrmrHarshShukla** [comment]
Hey @georgewrmarshall ,
Closing this one myself.
Equipped with all the experiences I gained from these PRs, I will be back soon with more organized and better PRs.

Regards,
Harsh

**Comment by georgewrmarshall** [comment]
Hey @PrgrmrHarshShukla, thanks again for all your contributions, best of luck and look forward to collaborating with you in future

**Pull request closed without merging**